libs/fluidsynth/src/fluid_rev.c

```diff
@@ -435,7 +435,7 @@ static void set_mod_frequency(sinus_modulator *mod,
 
     a = (2 * FLUID_M_PI / 360) * phase;
 
-    mod->buffer2 = FLUID_SIN(a - w); /* y(n-1) = sin(-intial angle) */
```

Comment:
This should go upstream to Fluidsynth. It will otherwise be overwritten when we sync libfluidsynth next time

Reply:
Submitted typo fix upstream in https://github.com/FluidSynth/fluidsynth/pull/1146  
Will revert this fix then

Reply:
squashed and rebased.